modules/statusline/lualine/config.nix

```diff
@@ -5,6 +5,7 @@
 }:
 with lib; let
   cfg = config.vim.statusline.lualine;
+  luaTable = items: ''{${builtins.concatStringsSep "," items}}'';
```

Comment:
I'd prefer if helper functions were in the extended library. This can go with test of the lua helpers.

Reply:
:+1:  there even is a listToLuaTable function already, though it's not exactly the right type here